### ISSUE TITLE
Race examples are outdated

### ISSUE THREAD

**Issue by jmakeig** [marklogic/entity-services#235]
After successfully completing the deployment I get the following report in the `01. load-report` tab of `races-qc.xml`. Are the target numbers wrong or has my set-up gone awry? 

```
Raw docs (1354 expected): 
1354
Non raw (1310 expected): 
6
Race envelopes (4 expected): 
0
Run envelopes (1300 expected): 
0
Collection names
csv
http://marklogic.com/entity-services/models
raw
reference
```

**Comment by jmakeig** [comment]
I just cleared my content database and am able to reproduce this. Am I missing a set-up step? I did:

```shell
cd entity-services-examples/
./gradlew -PexampleDir=example-races mlDeploy
./gradlew runExampleRaces
```

**Comment by bsrikan** [comment]
Setup step seems OK to me. I think the ingest transforms that generate envelopes is not working:

harmonizer.harmonize();
harmonizer.secondSourceHarmonize(); and 
translator.translate();

All those 1354 documents are raw source documents from ~/example-races/data directory

**Comment by bsrikan** [comment]
If you do run as:
../gradlew runExampleRaces

then everything seems OK. Looks like there is a difference in running ./gradlew and ../gradlew @jmakeig

**Comment by grechaw** [comment]
Actually there's a legit bug having to do with most recent changes--- DMSDK async finishes before harmonize is done.  I'll have a fix later today I think

**Comment by jmakeig** [comment]
> Actually there's a legit bug having to do with most recent changes--- DMSDK async finishes before harmonize is done.

Too legit to quit, apparently. (Sorry.)

**Comment by grechaw** [comment]
Very nice.  When I went through and updated the examples (because awaitTerminiation() had gone away), 
I used flushAsync everywhere in stead of flushAndWait.

I've taken out all the threading filigree.  Just running the three DMSDK jobs one after another is fine for the example.

**Comment by grechaw** [comment]
The other change is how Optic returns JSON.  I'm updating person example to have correct handling of these results, and I'm also going to make a qc workspace with the XQ and JS versions of those same optic queries.

**Comment by jmakeig** [comment]
Fixed by #236.
